LICENSE-binary

```diff
@@ -401,13 +428,15 @@ BSD 2-Clause
 ------------
 
 com.github.luben:zstd-jni
+dnsjava:dnsjava
```

Comment:
Likewise add copies of the licenses below to licenses-binary/

Reply:
> Likewise add copies of the licenses below to licenses-binary/

Done.